addon/-private/statechart/index.ts

```diff
@@ -0,0 +1,14 @@
+import { setComponentManager } from '@ember/component';
+
+import { StateMachine } from 'xstate';
+
+import ComponentManager from './manager';
+
+import type ApplicationInstance from '@ember/application/instance';
+
+setComponentManager(
+  (owner: ApplicationInstance) => {
+    return ComponentManager.create(owner);
+  } /* hmm, it seems XState creates a totally new object here, so we have no prototype or superclass to checkt */,
```

Comment:
@davidkpiano -- would it be possible to change how createMachine works so that it can return something with a common base class (or prototype). Component managers use (I think) `instanceof` to check how to render things, and since it seems `createMachine` returns a new vanilla object every time with no relation to other returned values of `createMachine`, we don't have a way to do this: https://github.com/NullVoxPopuli/ember-statechart-component/pull/2/files#diff-e0cec400eb23941bb9753886e0f6413c642dbbd0194a8dd66f01927eec161d2c